### PR TITLE
2.x

### PR DESCRIPTION
**Pull request by jarednova**
**Ticket**: #1580 

This will track the progress and tests for `2.x` into `master`

### CODE REVIEW

File: tests/test-timber-menu.php [hunk 1]
```diff
@@ -145,20 +145,56 @@ function testMenuTwigWithClasses() {
...
+		$item = $items[0];
+		$this->assertEquals( '_self', $item->target() );
+
+		// Menu item with _menu_item_target set to '_blank'
+		$item = $items[1];
+		$this->assertEquals( '_blank', $item->target() );
+
+		// Menu item with _menu_item_target set to ''
+		$item = $items[2];
+		$this->assertFalse( '_self', $item->target() );
```
Comment (jarednova) [inline tests/test-timber-menu.php]: @gchtr wanted to check on this test (it's failing right now). Should `$item->target()` register as `_self` or `false` in this scenario?

File: tests/test-timber-post.php [hunk 2]
```diff
@@ -379,6 +379,47 @@ function testPagedContent(){
 			$this->assertEquals($page2, trim(strip_tags( $post->paged_content() )));
 		}
 
+		function testMetaCustomPreFilterDisable(){
```
Comment (gchtr) [inline tests/test-timber-post.php]: @jarednova These test show a few errors with 2.x. I’ll make a new pull request to fix them.

Comment (jarednova) [reply tests/test-timber-post.php]: Blast! I was afraid that might happen

File: tests/test-timber-menu.php (hunk 1 above)
Comment (gchtr) [reply tests/test-timber-menu.php]: @jarednova See #2019.

**Comment by szepeviktor** [comment]
What a long labor!

**Comment by jarednova** [comment]
but a labor of love!

**Comment by jarednova** [comment]
@szepeviktor nope, that's an oversight on my part. There was a bad merge conflict I was trying to untangle so I had temporarily disabled. It's now back, thanks for the alert!

**Comment by gchtr** [comment]
Timber 2.0 is released: https://github.com/timber/timber/releases/tag/2.0.0.

We kept the `2.x` branch and made it the default branch. We’re not going to merge this into `master` (which was renamed to `1.x`). So we can close this pull request.

**Comment by szepeviktor** [comment]
One major version bump of Timber equals 5 children from a female 👱🏻‍♀️

**Pull request closed without merging**